src/openforms/submissions/api/validation.py

```diff
@@ -81,6 +90,12 @@ def get_submission_completion_serializer(
             "incomplete_steps": incomplete_steps,
             "submission_allowed": submission.form.submission_allowed,
             "privacy_policy_accepted": request.data.get("privacy_policy_accepted"),
+            "contains_blocked_steps": any(
+                [
+                    not submission_step.can_submit
+                    for submission_step in state.submission_steps
+                ]
+            ),
```

Comment:
```suggestion
            "contains_blocked_steps": any(
                    not submission_step.can_submit
                    for submission_step in state.submission_steps
            ),
```
not sure what black says about the formatting, but the list creation is needless.